spec/models/proposal_spec.rb

```diff
@@ -332,4 +332,13 @@
       expect(observation_creator_double).to have_received(:run)
     end
   end
+
+  describe "#tags" do
+    it "can add case-insensitive tags" do
```

Comment:
The README claims:

If you want to change the default delimiter (it defaults to ','). You can also pass in an array of delimiters such as ([',', '|']):
```
ActsAsTaggableOn.delimiter = ','
```

There's no mention of whitespace, and we haven't deviated from the default, so do we need to test that feature?